### PR TITLE
Create simple website

### PR DESCRIPTION
**Pull request by matankb**
We just need something that looks semi-professional to submit for Slack verification. Obviously we'll want something better in the long run, but not super urgently.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+.DS_Store
```
Comment (jdtzmn) [inline .gitignore]: Having a newline here is important to the upmost

File: index.html [hunk 2]
```diff
@@ -1,9 +1,193 @@
...
+      <div class="product more-soon">
+        <div class="">More coming soon!</div>
+      </div>
+    </div>
+
+    <div class="team-wrap">
+      <h2>Our Team</h2>
+      <div class="team">
+        <div class="team-member">
+          <img src="/assets/profiles/jacob.jpg" alt="Matan" />
```
Comment (jdtzmn) [inline index.html]: Alt should be Jacob

File: index.html [hunk 3]
```diff
@@ -1,9 +1,193 @@
...
+          <h3>Jacob Daitzman</h3>
+          <p>Co-Founder</p>
+        </div>
+        <div class="team-member">
+          <img src="/assets/profiles/shai.jpg" alt="Matan" />
```
Comment (jdtzmn) [inline index.html]: Alt should be Shai

File: index.html [hunk 4]
```diff
@@ -1,9 +1,193 @@
...
+            href="http://restore.ghostsoftware.co/"
+            rel="noopener noreferrer"
+            target="_blank"
+            >Join Waitlist</a
+          >
+        </div>
+      </div>
+
+      <div class="product more-soon">
+        <div class="">More coming soon!</div>
```
Comment (jdtzmn) [inline index.html]: Empty class

**Review by jdtzmn — APPROVED** [review]

File: .gitignore (hunk 1 above)
Comment (matankb) [reply .gitignore]: i'm gonna kill you

**Pull request merged**